styles.css

```diff
@@ -132,6 +133,209 @@ body {
+        row-gap: 88px;
+    }
+
+    .grid-item {
+        display: flex;
+        flex-direction: column;
+        border-radius: 16px;
+        border: 1px solid #dfe1e6;
+        background-color: #FFFFFF;
+    }
+
+    .grid-item img {
+        display: flex;
+        justify-content: center;
+        margin: 16px;
+    }
+
+    .grid-item h2 {
+        display: flex;
+        font-size: 32px;
+        line-height: 44px;
+        font-weight: 700;
+        margin: -3px 16px;
+    }
+
+    .details {
+        display: flex;
+        font-size: 13px;
+        line-height: 16px;
+        font-weight: 600;
+        margin: 0 16px;
+    }
+
+    .details ul {
+        margin: 0;
+        padding: 0;
+    }
+
+    .details li {
+        list-style-type: none;
+        display: flex;
+        align-items: center;
+        margin-top: 7px;
+    }
+
+    .details img {
+        margin: 16px 11px;
+    }
+
+    .worksText {
+        font-size: 15px;
+        line-height: 24px;
+        font-weight: 400;
+        margin: 8px 16px;
+    }
+
+    .worksTag {
+        font-size: 12px;
+        line-height: 16px;
+        font-weight: 500;
+    }
+
+    .worksTag ul {
+        margin: 2px 16px;
+        padding: 0;
+        display: flex;
+
+    }
+
+    .worksTag li {
+        list-style-type: none;
+        background-color:#EBEBFF;
+        padding: 4px 12px;
+        margin-right: 8px;
+        margin-top: 5px;
+        border-radius: 8px;
+        color: #6070FF;
+    }
+
+    .worksButton {
+        width: 124px;
+        height: 48px;
+        margin: 20px 16px;
+        font-size: 17px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+
+    .aboutMe {
+        padding: 114px 24px;
+    }
+
+    .aboutMe h1 {
+        font-size: 40px;
+        line-height: 52px;
+        font-weight: 700;
+        margin-top: 0;
+    }
+
+    .aboutPara {
+        margin-top: -7%;
+        font-size: 15px;
+        line-height: 24px;
+    }
+
+    .aboutLink {
+        margin-top: -2.5%;
+        font-size: 16px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #7F8CFF;
+    }
+
+    .aboutMe li {
+        list-style-type: none;
+        margin-left: -13.5%;
+        padding-top: 5px;
+    }
+
+    .aboutMe img {
+        height: 18px;
+        margin-right: 23px;
+    }
+
+    .aboutMeButton {
+        width: 162px;
+        height: 48px;
+        margin: 0px;
+        padding: 12px;
+        font-size: 17px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+    
+    .aboutMeDropdown {
+        margin: 0;
+        padding: 0;
+        display: flex;
+    }
+
+    .dropdownItem {
+        font-size: 20px;
+        line-height: 24px;
+        font-weight: 500;
+    }
```

Comment:
- [ ] For the abilities section please consider adding the icons for the languages, skills, and frameworks so it looks similar to the Figma design.

### Your design:
![image](https://user-images.githubusercontent.com/68030297/219881914-86cc8b35-7a41-41a0-b86f-bcdd9bc2b084.png)
![image](https://user-images.githubusercontent.com/68030297/219881894-c918e4d3-351c-497b-91e8-a4e444cb11f1.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219881956-7f0c6f16-31c9-4c39-9891-2be5bc343865.png)
![image](https://user-images.githubusercontent.com/68030297/219881962-bbf767c1-710e-4464-bec0-504b62380085.png)